docs/multi-cluster-mesh/active-passive/multi-cluster-active-passive.md

```diff
@@ -0,0 +1,106 @@
+# Single mesh multi cluster multiple networks with Istio operator
+
+## Objective:
+Setup three clusters (one active – two passive) in a single mesh multi network (two in the same network, one in a different network) setup with Istio operator.
+
+## Setup: 
+
+### Steps:
+
+#### Active setup:
+1. Install cluster registry controller:
+```
+helm install --namespace=cluster-registry --create-namespace cluster-registry-controller deploy/charts/cluster-registry  --set image.tag=v0.1.9 --set localCluster.name=demo-active --set network.name=network1
```

Comment:
Pls use sth like this: https://github.com/cisco-open/cluster-registry-controller/tree/master/deploy/charts/cluster-registry#installing-the-chart

So use publicly available helm charts instead of local helm chart paths.

Reply:
pls change this